code/_globalvars/traits.dm

```diff
@@ -111,6 +111,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_NICE_SHOT" = TRAIT_NICE_SHOT,
 		"TRAIT_ALWAYS_STUBS" = TRAIT_ALWAYS_STUBS,
 		"TRAIT_NAIVE" = TRAIT_NAIVE,
+		"TRAIT_BLOODSHOT_EYES" = TRAIT_BLOODSHOT_EYES,
```

Comment:
Shouldnt TRAIT_ANTICONVULSANT  also be in here?

Reply:
sure. Thats the list that admins can use to add from, right?

Reply:
added in 0477259